js/main.js

```diff
@@ -18,4 +19,91 @@ $entryForm.addEventListener('submit', function submitForm(event) {
   data.nextEntryId++;
   $img.setAttribute('src', './images/placeholder-image-square.jpg');
   $entryForm.reset();
+
+  const $newDomTree = renderEntry(entryData);
+  const $unorderedList = document.querySelector('ul');
```

Comment:
`querySelector`s should always be done globally. When you do them inside of functions, you're forcing us to traverse the DOM tree over and over. Eventually you are going to need this element again, so move this to the global space